notificationservice.cpp

```diff
@@ -46,6 +46,7 @@ public slots:
                     else if(mReader.name() == "ai") appIcon = mReader.readElementText();
                     else if(mReader.name() == "su") summary = mReader.readElementText();
                     else if(mReader.name() == "bo") body = mReader.readElementText();
+                    else if(mReader.name() == "vb") vibrate = mReader.readElementText();
```

Comment:
I wanted to keep the terminology here close to the one found in user facing strings in the android app. I'm open to changing them in both places now: what do you suggest?